content/250-postgres/1200-more/1000-faq.mdx

```diff
@@ -73,6 +75,148 @@ Yes, read and write queries are counted equally as _operations_ and are billed t
+
+:::
+
+We will associate each workload with a plan and its corresponding pricing details, for example, the **Starter plan** for the small workload, the **Pro plan** for the medium workload, and the **Business Annual plan** for the large workload. Then we will apply the equations to the example workloads to generate a rough estimate of a monthly bill. For example:
+
+:::note[Pricing details]
+
+Here are the details for each pricing plan:
+
+- **Starter plan** - \$18 per million operations
+  - Base plan fee - \$0 per month
+  - Storage - 1 GB free then \$2 per additional GB
+- **Pro plan** - \$8 per million operations
+  - Base plan fee - \$49.00 per month
+  - Storage - 5 GB free then \$1.5 per additional GB
+- **Business Annual plan** - \$4 per million operations in the 50-100M/month usage bracket
+  - Base plan fee - \$129.00 per month
+  - Storage - 10 GB free then \$1 per additional GB
+
+You can also learn more about the pricing details for each plan on the [pricing page](https://www.prisma.io/pricing?utm_source=docs).
+
+:::
+
+**Example of a small workload on the Starter plan**:
+
+A hobby or early-stage side-project with ~`500` MAUs. Each user clicks around `10` times a day, and the entire database uses about `0.5` GB of storage. Based on the assumptions, you would calculate the monthly bill using the following equations:
+
+- `total_ops` = `500` x `10` x `30` = `150000`
+- `billable_ops` = `50000`
+- `ops_cost` = (`50000` ÷ `1000000`) x $`18` = ~$`0.90`
+- `storage_cost` = 0 GB (0.5 GB is below the 1 GB free tier)
+- `base_plan_fee` = $`0`
+
+`total_monthly_cost` = ~$`0.90` per month
+
+**Example of a medium workload on the Pro plan**:
+
+A growing SaaS product serving ~`5000` MAUs. Power users average `40` actions per day, and the app stores roughly `6` GB of data. Based on the assumptions, you would calculate the monthly bill using the following equations:
+
+- `total_ops` = `5000` x `40` x `30` = `6000000`
+- `billable_ops` = `5900000`
+- `ops_cost` = (`5900000` ÷ `1000000`) = `5.9` x $`8` = ~$`47.20`
+- `storage_cost` = (`6` GB - `5` GB) x $`1.50` = $`1.50`
+- `base_plan_fee` = $`49.00`
+
+`total_monthly_cost` = $`47.20` + `1.50` + `49.00` = ~$`97.70`  per month
+
+**Example of a large workload on the Business Annual plan**:
+
+A production-grade, consumer-facing application handling ~`50000` MAUs. Heavy usage with `60` actions per user per day drives significant traffic, and the dataset reaches ~`40` GB.  Based on the assumptions, you would calculate the monthly bill using the following equations:
+
+- `total_ops` = `50000` x `60` x `30` = `90000000`
+- `billable_ops` = `89900000`
+- `ops_cost` = (`89900000` ÷ `1000000`) = `89.9` x $`4` = ~$`359.6`
+- `storage_cost` = (`40` GB - `10` GB) x $`1` = $`30.00`
+- `base_plan_fee` = $`129.00`
+
+`total_monthly_cost` = $`359.6` + `30.00` + `129.00` = ~$`518.60` per month
```

Comment:
```suggestion
`total_monthly_cost` = $`359.6` + $`30.00` + $`129.00` = ~$`518.60` per month
```

Reply:
where we're doing calculations, the "~" doesn't really make sense. when adding 1 and 1 together, it's not approx (~) 2, it's exactly 2. Can you revise where and how ~ is used based on this?